src/logging/logger.ts

```diff
@@ -333,22 +357,25 @@ export class Logger implements ZLUX.Logger {
     return false;
   }
 
-  makeComponentLogger(componentName:string):ComponentLogger{
-    let componentLogger:ComponentLogger|undefined = this.componentLoggers.get(componentName);
+  makeComponentLogger(componentName:string, messages?: any):ComponentLogger{
+    let componentLogger:ComponentLogger|undefined = componentLoggers.get(componentName);
     if (componentLogger){
       this.consoleLogInternal("_internal",LogLevel.WARN,
                               `${Logger.processString}${Logger.username} ${LogLevel[1]}`,
                               true,true,
                               'Logger created with identical component name to pre-existing logger. Messages overlap may occur.');
     } else {
-      componentLogger = new ComponentLogger(this,componentName);
+      componentLogger = new ComponentLogger(this, componentName);
```

Comment:
Stylistically it is better to have messages be a 3rd arg here than to assign them directly.

Reply:
Fixed, makes sense